osidb/tests/test_search.py

```diff
@@ -227,3 +229,13 @@ def test_search_flaws_on_particular_columns(self, auth_client, test_api_uri):
         assert response.status_code == 200
         body = response.json()
         assert body["count"] == 1
+
+        response = auth_client().get(f"{test_api_uri}/flaws?team_id=1234")
+        assert response.status_code == 200
+        body = response.json()
+        assert body["count"] == 1
```

Comment:
this doesn't really test the filter, if the filter is not valid this will also be `True`, you need more than one flaw and to check that the team_id corresponds

Reply:
you need to check that `body["team_id"]` matches what you expect, if someone modifies the filter down the line and the filter returns the same number of records but the incorrect ones, you won't know it's broken, same for all the filters below